### ISSUE TITLE
Uncaught (in promise) TypeError: Illegal invocation

### ISSUE THREAD

**Issue by njakob** [marvinroger/nanocurrency-js#5]
Thank you for the great work!

While trying out your library in run into the following error in Chrome, while using workers:

```
nanocurrency.umd.js:6 Uncaught (in promise) TypeError: Illegal invocation
```

You can reproduce the error by using the following in your multi-threaded example:

```js
await NanoCurrency.init()
const work = await NanoCurrency.generateSeed()
```

Seems the error comes from [here](https://github.com/marvinroger/nanocurrency-js/blob/master/src/index.js#L182).

Manage to get around by generating the seed myself:

```js
const array = new Uint8Array(32);
self.crypto.getRandomValues(array);
const seed = array.reduce((hex, idx) => {
  return hex + ('0' + idx.toString(16)).slice(-2);
}, '');
```

**Comment by marvinroger** [comment]
Thanks for the report! A simple `bind` should fix that, I will do it today.

**Comment by marvinroger** [comment]
Fixed in v1.6.1.
